pom.xml

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.hotels</groupId>
+    <artifactId>hotels-oss-parent</artifactId>
+    <version>2.0.5</version>
+  </parent>
+
+  <groupId>com.hotels</groupId>
+  <artifactId>housekeeping</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <inceptionYear>2017</inceptionYear>
+
+  <scm>
+    <connection>scm:git:git@github.com:HotelsDotCom/housekeeping.git</connection>
+    <developerConnection>scm:git:git@github.com:HotelsDotCom/housekeeping.git</developerConnection>
+    <url>https://github.com/HotelsDotCom/housekeeping</url>
+    <tag>HEAD</tag>
+  </scm>
+
+  <properties>
+    <powermock.version>1.6.4</powermock.version>
+    <mockito.version>1.10.19</mockito.version>
+
+    <jasypt-spring-boot.version>1.5-java7</jasypt-spring-boot.version>
+    <hadoop.version>2.7.1</hadoop.version>
+  </properties>
+
+  <dependencies>
+    <!-- Hadoop -->
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-common</artifactId>
+      <version>${hadoop.version}</version>
+      <scope>provided</scope>
+    </dependency>
+
+    <!-- DataSource (HikariCP) -->
+    <dependency>
+      <groupId>com.zaxxer</groupId>
+      <artifactId>HikariCP</artifactId>
+      <version>2.4.3</version>
+    </dependency>
+
+    <dependency>
+      <groupId>joda-time</groupId>
+      <artifactId>joda-time</artifactId>
+      <version>2.8.2</version>
+    </dependency>
+
+    <!-- Hibernate -->
+    <dependency>
+      <groupId>org.hibernate</groupId>
+      <artifactId>hibernate-entitymanager</artifactId>
+      <version>4.3.11.Final</version>
```

Comment:
use a POM property

Reply:
Yeah, I think anything that is repeated more than once warrants a property so if we change the version we only have to do it in one place.